### PR TITLE
Improve inline/iframe rendering documentation

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

- [x] Clarify how to opt-in to inline rendering in Angular
- [x] Separate out the custom `prepareForInline` docs, which are advanced usage

Self-merging @jonniebigodes 

## How to test

N/A

### CODE REVIEW

File: docs/writing-docs/docs-page.md [hunk 1]
```diff
@@ -168,11 +168,26 @@ Unless you use a custom [webpack configuration](../configure/webpack.md#extendin
...
+};
+```
 
-The iframe creates a clean separation between your code and Storybook’s UI. But using an iframe has disadvantages. For example, you have to set the height of iframe stories explicitly, or you’ll see a scroll bar. And certain dev tools might not work right.
+### Custom inline rendering
 
-Render your framework’s stories inline using two docs configuration options in tandem, `inlineStories` and `prepareForInline`.
+If your framework doesn't [support inline rendering](../api/frameworks-feature-support.md), you also need to provide a `prepareForInline` function in addition to the `inlineStories` parameter.
 
 Setting `inlineStories` to `true` tells Storybook to stop putting your stories in an iframe. The `prepareForInline` accepts a function that transforms story content from your given framework to something React can render (Storybook’s UI is built in React).
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply docs/writing-docs/docs-page.md:192]: @shilman indeed we do. I'll follow up with you async on this and we'll go from there! @snowfrogdev appreciate the feedback you added to this and willingness to polish the documentation !

[the remark this answers is not in the document]
Comment (snowfrogdev) [reply docs/writing-docs/docs-page.md:192]: > @snowfrogdev Angular already has a `prepareForInline` built in, so you don't need to provide any yourself, you just need to set `inlineStories: true`.

Then there is something weird going on. In my project, `inlineStories` is set to `true`, and was set to that from the beginning, on sb installation, yet my components are still rendered in an iframe. Could I be missing something?

[the remark this answers is not in the document]
Comment (shilman) [reply docs/writing-docs/docs-page.md:192]: Which version are you running?

[the remark this answers is not in the document]
Comment (snowfrogdev) [reply docs/writing-docs/docs-page.md:192]: > Which version are you running?

6.3.12

image

**Pull request merged**